src/xcode/ENA/ENA/Source/Workers/Store.swift

```diff
@@ -224,6 +227,11 @@ final class SecureStore: Store {
 		set { kvStore["hourlyFetchingEnabled"] = newValue }
 	}
 
+	var previousRisk: Risk? {
```

Comment:
@llanghammer-sap I think transactions become more and more unavoidable :D Sorry. Let's talk about it once more - maybe tomorrow.